MapleServer2/Packets/ServerEnterPacket.cs

```diff
@@ -63,4 +63,4 @@ public static class ServerEnterPacket {
             return PacketWriter.Of(SendOp.FINALIZE_SERVER_ENTER);
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
When you're working with a new file, `CTRL+K` then `CTRL+D` to  format style

Reply:
is an existing file.. i actually only added "session.Player.Merets"